README.md

```diff
@@ -18,34 +18,15 @@
 
     gem install rspec-puppet
 
-> Note for ruby 1.8 users:  while rspec-puppet itself supports ruby 1.8, you'll
-> need to pin rspec itself to `~> 3.1.0`, as later rspec versions do not work
-> on old rubies anymore.
-
 ## Starting out with a new module
 
 When you start out on a new module, create a metadata.json file for your module and then run `rspec-puppet-init` to create the necessary files to configure rspec-puppet for your module's tests.
 
-
-## Configure manifests for Puppet 4
-
-With Puppet 3, the manifest is set to `$manifestdir/site.pp`. However Puppet 4 defaults to an empty value. In order to test manifests you will need to set appropriate settings.
-
-Puppet configuration reference for `manifest` can be found online:
-
-* Puppet 3: https://docs.puppet.com/puppet/3.8/reference/configuration.html#manifest
-* Puppet 4: https://docs.puppet.com/puppet/4.8/reference/configuration.html#manifest
-
 Configuration is typically done in a `spec/spec_helper.rb` file which each of your spec will require. Example code:
 ```ruby
-# /spec
-base_dir = File.dirname(File.expand_path(__FILE__))
-
 RSpec.configure do |c|
-  c.module_path     = File.join(base_dir, 'fixtures', 'modules')
-  c.manifest_dir    = File.join(base_dir, 'fixtures', 'manifests')
-  c.manifest        = File.join(base_dir, 'fixtures', 'manifests', 'site.pp')
```

Comment:
We utilize the `manifest` setting when testing our control repository to ensure anything in the environment `site.pp` is included during tests as shown in the `spec/spec_helper.rb` fragment below. Is there a better way to accomplish this?
```ruby
RSpec.configure do |c|
  c.manifest = File.join(File.dirname(File.dirname(__FILE__)), 'manifests', 'site.pp')
end
```

Reply:
The reason I dropped it is that the `manifest` setting is gone. At least in https://www.puppet.com/docs/puppet/7/configuration.html I can no longer find it. I suspect environments now determine this via `environment.conf`. But feedback is welcome. I may have missed something.

Reply:
I'm pretty sure I ran into a situation myself where I needed to the manifest, so I think you're right that this should remain.

Reply:
It might be the case that Puppet doesn't use this setting directly anymore, but it is used by rspec-puppet internally to prepend additional Puppet code to test:
https://github.com/puppetlabs/rspec-puppet/blob/d21b57039c000bd0a6ea5d441e40cab97593e6e6/lib/rspec-puppet/support.rb#L25
https://github.com/puppetlabs/rspec-puppet/blob/d21b57039c000bd0a6ea5d441e40cab97593e6e6/lib/rspec-puppet/support.rb#L153-L166

Reply:
Yes, I found this in our Infra repo where I use it to test our environment, rather than individual modules: https://github.com/theforeman/foreman-infra/blob/9d05130a5ada4fd734898b6e14b93bd292c92717/puppet/spec/spec_helper.rb#L29